public/index.html

```diff
@@ -50,7 +49,17 @@ <h4> Create a Geolocating Web Annotation </h4>
                 
             </div>
         </div>
+
+        <div class="navLinks">
```

Comment:
Since you want this on every page, make a custom element for it.  See See `GeoPage`, `GeoHeader`, `GeoFooter` in [layout.js](https://github.com/CenterForDigitalHumanities/geolocator/blob/siteNav/public/components/layout.js#L10).